README.md

```diff
@@ -164,6 +164,25 @@ document.getElementById('logout').addEventListener('click', () => {
 });
 ```
 
+### Caching strategy
+
+The SDK can be configured to cache data either in memory or in local storage. The default is in memory. This setting can be controlled using the `cacheStrategy` option when creating the Auth0 client.
```

Comment:
I think we should clarify what we mean by "data" up-front here to reference ID and access tokens.